### ISSUE TITLE
Could not inspect Wpf element using FlaUI

### ISSUE THREAD

**Issue by bawarim** [FlaUI/FlaUI#496]
**Describe the bug**
I am trying to inspect an WPF element inside System.Windows.Controls.Grid using FlaUI, but it is not getting listed on object tree. 
The same object I have tried to identify with UFT and there it is getting identified.

So could you please suggest what changes do we need to do so that the element can be identified by FlaUI as well. 

**Screenshots**
This is how it is getting identified using UFT object spy.
![image](https://user-images.githubusercontent.com/46487726/174764300-8bf67048-9187-4928-be7c-b9682ef31baa.png)

Here, element is not getting listed in the tree.
![image](https://user-images.githubusercontent.com/46487726/174764728-63184748-366f-4dd4-ba67-3a166723f002.png)

**Comment by Roemer** [comment]
UFT seems to use the logical WPF tree to find this element, FlaUI (currently) solely relies on UIA. And for UIA, it is the duty of the WPF controls to correctly provide all elements. I guess you're using Infragistics, so you should inform them, that this button is not exposed to UIA.

**Comment by bawarim** [comment]
@Roemer Yes, its an Infragistics control. So is it possible to expose it to UIA? Could you please suggest any document to do the same. I have checked with my dev team, but they don't have any idea for the same.
